src/Components/SourcesSelect/index.js

```diff
@@ -63,22 +65,38 @@ const SourcesSelect = ({ setValidation, image }) => {
   if (isLoading) {
     return <Spinner isSVG size="sm" aria-label="Loading accounts" />;
   }
+  const isSelectedSourceUnavailable = selected != undefined && sources.find((source) => source.id === selected.id && source.status != 'available');
+
+  let link = isBeta() ? '/preview/settings/sources/' : '/settings/sources/';
+  if (chosenSource) {
+    link = isBeta() ? `/preview/settings/sources/detail/${chosenSource}` : `/settings/sources/detail/${chosenSource}`;
+  }
 
   return (
-    <Select
-      ouiaId="select_account"
-      isOpen={isOpen}
-      onToggle={(openState) => setIsOpen(openState)}
-      selections={selected}
-      maxHeight="180px"
-      // TODO decide if to disable the select
-      // isDisabled={sources?.length === 1}
-      onSelect={onSelect}
-      placeholderText="Select account"
-      aria-label="Select account"
-    >
-      {sources && selectItemsMapper()}
-    </Select>
+    <>
+      <Select
+        ouiaId="select_account"
+        isOpen={isOpen}
+        onToggle={(openState) => setIsOpen(openState)}
+        selections={selected}
+        maxHeight="180px"
+        validated={isSelectedSourceUnavailable && 'warning'}
+        // TODO decide if to disable the select
+        // isDisabled={sources?.length === 1}
+        onSelect={onSelect}
+        placeholderText="Select account"
+        aria-label="Select account"
+      >
+        {sources && selectItemsMapper()}
+      </Select>
+      {isSelectedSourceUnavailable && (
+        <HelperText id="account-error-inline">
+          <HelperTextItem variant="warning">
+            source is unavailable, for more information click <a href={link}> here </a>
```

Comment:
do we want to open the link in a new window?

Reply:
Opened it in a new tab , the linter added the spaces like that.